docs/collections/return-data-in-batches.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Returning Collection Data
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The Macrometa GDN is a multi-tenant platform for Free and Metered tier users. There are default limits for both tiers detailed in [Tenant Quotas and Limits](../references/quotas.md). This page describes some options to retrieve larger result sets from a collection despite the default limit of 1,000 documents per query.
+
+## SDK download
+
+:::note
+This solution is not currently supported in the pyc8 SDK
+:::
+
+<Tabs groupId="operating-systems">
+<TabItem value="js" label="Javascript">
+
+```js
+With Yarn or NPM
+
+    yarn add jsc8@beta
+    (or)
+    npm install jsc8@beta
+
+If you want to use the SDK outside of the current directory, you can also install it globally using the `--global` flag:
+
+    npm install --global jsc8@beta
+
+From source,
+
+    git clone https://github.com/macrometacorp/jsc8.git
+    cd jsC8
+    npm install
+    npm run dist
+```
+</TabItem>
+</Tabs>
+
+## Solutions
+
+There are two recommmended approaches depending on the volume of data being returned. The Cursor API can be used to fetch up to 30,000 records, this is an approximation based on platform limits. Attempting to return more than 30,000 from the Cursor API may cause the query to exceed the query execution time limit and raise an error. The Collection Export API is the recommended approach when retrieving an entire collection.
+
+Create a connection to the GDN using the jsc8 SDK. You will need an API key, JWT, or Email/Password to complete authentication.
```

Comment:
```suggestion
Create a connection to the GDN using the jsc8 SDK. You will need an API key, JWT, or Email/Password to complete authentication. For more information, refer to [Authentication](link to authentication topic)
```

Reply:
Notice that you need to add the link path here.

Reply:
Added, thank you!